### PR TITLE
Fix `Doc.from_docs` for `doc.ents`

### PR DESCRIPTION
**Pull request by tamuhey**
## Description

`Doc.from_docs` doesn't merge `doc.ents` properly.

### Types of change

Bug fix

## Checklist

- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/tokens/doc.pyx [hunk 1]
```diff
@@ -884,7 +884,7 @@ cdef class Doc:
 
         if attrs is None:
             attrs = [LEMMA, NORM]
-            if all(doc.is_nered for doc in docs):
+            if any(doc.is_nered for doc in docs):
                 attrs.extend([ENT_IOB, ENT_KB_ID, ENT_TYPE])
```
Comment (svlandeg) [inline spacy/tokens/doc.pyx:888]: Agreed that `any` makes more sense! What about the cases for `is_tagged` and `is_parsed` just a few lines below this one - shouldn't they be fixed in a similar fashion too?

**Review by svlandeg — APPROVED** [review]
Hi @tamuhey, thanks for the PR(s)! `Doc.from_docs` is new in v.3, and apparently requires more testing, so it's great to add more tests to the suite. 

We definitely appreciate you hunting bugs even before the release :-)

**Comment by adrianeboyd** [comment]
Since there were a few related problems with these attributes, I tried to address this more generally in #5962. If that PR is merged with the relevant changes, then this one can be closed, I think.

**Comment by tamuhey** [comment]
@adrianeboyd Cool! Why don't you add the test of this PR to #5962 ?

**Comment by adrianeboyd** [comment]
Good suggestion, thanks!

**Comment by adrianeboyd** [comment]
Okay, I think this is fixed by #6044.

**Pull request closed without merging**